User/UserRepositoryCollection.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace SumoCoders\FrameworkMultiUserBundle\User;
+
+use Doctrine\Common\Collections\ArrayCollection;
+use SumoCoders\FrameworkMultiUserBundle\Exception\RepositoryNotRegisteredException;
+
+class UserRepositoryCollection
+{
+    /**
+     * @var ArrayCollection
+     */
+    private $userRepositories;
+
+    /**
+     * UserRepositoryCollection constructor.
+     *
+     * @param array $userRepositories
+     */
+    public function __construct(array $userRepositories)
+    {
+        $this->userRepositories = new ArrayCollection();
```

Comment:
wa is de meerwaarde hier om er een collection van te maken? Je gebruikt geen enkele collectie specifieke method. Dan kan je evenwel een array gebruiken.